app/views/courses/modern_languages/edit.erb

```diff
@@ -0,0 +1,39 @@
+<%= content_for :page_title, "Change modern languages – #{course.name_and_code}" %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(details_provider_recruitment_cycle_course_path(course.provider_code, course.recruitment_cycle_year, course.course_code)) %>
+<% end %>
+
+<%= render 'shared/errors' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: course,
+          url: modern_languages_provider_recruitment_cycle_course_path(@course.provider_code, @course.recruitment_cycle_year, @course.course_code),
+          method: :put do |form| %>
+      <div
+        class="<%= classnames("govuk-form-group", "govuk-form-group--error": @errors && @errors[:subjects]&.any?) %>"
+        data-qa="course__modern_languages_section">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+            <h1 class="govuk-fieldset__heading">
+              <span class="govuk-caption-xl"><%= course.name_and_code %></span>
+              Change modern languages
+            </h1>
+          </legend>
+          <%= render "shared/error_messages", field: :subjects if @errors %>
+          <%= render 'form_fields', form: form %>
```

Comment:
it's ambiguous which `form_fields` is being referred to as there are many in our code

can we do the same as https://github.com/DFE-Digital/manage-courses-frontend/blob/5f1a40fd1b065a963da98d2e90b6b18a752ff51b/app/views/courses/level/new.html.erb#L29 ?